packages/editor/src/components/autosave-monitor/index.native.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { compose } from '@wordpress/compose';
+import { withSelect } from '@wordpress/data';
+
+/**
+ * External dependencies
+ */
+import RNReactNativeGutenbergBridge from 'react-native-gutenberg-bridge';
```

Comment:
Can we have a `editorDidAutosave()` method on `react-native-gutenberg-bridge` and import it here?
Is good to have all the bridge functionality declared on `react-native-gutenberg-bridge/index.js`, so we can keep track of its functionality.